src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-base/index.js

```diff
@@ -725,5 +725,9 @@ export default {
         isFavorite() {
             return this.salesChannelFavoritesService.isFavorite(this.salesChannel.id);
         },
+
+        validateMaintenanceIpCidr(term) {
```

Comment:
Please add a test case

Reply:
Is it easier now to run vue tests?

Reply:
If you are referring to the transition from Vue 2 to Vue 3, then yes. However, in a broader context, there haven't been significant updates in this regard. Specifically in your situation, the task should be relatively straightforward. The `string.utils.ts` file is not a component (refer to something similar, like [types.utils.spec.js](https://github.com/shopware/shopware/blob/trunk/src/Administration/Resources/app/administration/src/core/service/utils/types.utils.spec.js)), and the `sw-sales-channel-detail` component already has a good test.